cores/swerv.core

```diff
@@ -100,6 +100,8 @@ targets:
       rivierapro:
         vlog_options :
           - config/common_defines.vh
+        compilation_mode:
+          - common
```

Comment:
compilation mode is defined as string, not a list, so this should be `compilation_mode : common`

Reply:
ok, sorry my bad